__tests__/app.test.js

```diff
@@ -42,3 +42,38 @@ describe('/api', () => {
       });
   });
 });
+describe('/api/articles/:article_id', () => {
+  test('get object describing an article using specific id', () => {
+    return request(app)
+      .get('/api/articles/3')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.author).toBe('icellusedkars');
+        expect(body.title).toBe('Eight pug gifs that remind me of mitch');
+        expect(body.article_id).toBe(3);
+        expect(body.body).toBe('some gifs');
+        expect(body.topic).toBe('mitch');
+        expect(body.created_at).toBe('2020-11-03T09:12:00.000Z');
+        expect(body.votes).toBe(0);
+        expect(body.article_img_url).toBe(
+          'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        );
+      });
+  });
+  test('responds with msg article does not exists if providet with valid but non existent id', () => {
+    return request(app)
+      .get('/api/articles/3333')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('article does not exist');
+      });
+  });
+  test('responds with an appropriate status and error message when given an invalid id', () => {
+    return request(app)
+      .get('/api/articles/nonsens')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad request');
+      });
+  });
```

Comment:
Excellent, well done for testing both an valid ID which does not exist and an invalid ID. You have responded with the correct status codes.

Some tips here:

- same feedback as above, which is to add the expected status code in the test description
- be careful of spelling mistakes in your test descriptions